package.json

```diff
@@ -113,6 +126,9 @@
         "start-80": "sudo gatsby develop -p 80",
         "serve": "gatsby serve",
         "test": "npm run eslint && npm run stylelint",
+        "jest": "jest --passWithNoTests --detectOpenHandles",
+        "jest:coverage": "npm run jest -- --coverage",
+        "jest:watch": "npm run jest -- --watch",
```

Comment:
what is that extra `--` in this script?

Reply:
@ali-hosseini-fs It's for passing arguments. 
please check this link: https://docs.npmjs.com/cli/v8/commands/npm-run-script

Reply:
without it the coverage won't work. I did some search and according to this [link](https://stackoverflow.com/questions/24825860/how-to-get-the-code-coverage-report-using-jest) I found the issue.